src/request/payload/event.ts

```diff
@@ -948,6 +948,7 @@ export interface UserProfileChangedEvent
     team_id: string;
     name: string;
     deleted: boolean;
+    is_forgotten: boolean;
```

Comment:
It seems that this flag does not always appear:
```suggestion
    is_forgotten?: boolean;
```